default mapping: Y => y$ (#13268)
neovim/neovim: runtime/doc/change.txt

["x]yy			Yank [count] lines [into register x] |linewise|.

							*Y*

> ["x]Y			Yank [count] lines from the cursor to the end of line.

Reviewer: The doc for `:help Y` would not change, rather the pattern would be something like this: 

1. mention the default mapping next to the (legacy) builtin.
2. add a tag like `*Y-default*`

```
*Y-default*
This command is mapped to "y$" by default. |nvim-default-mappings|
```

							*Y*
["x]Y			yank [count] lines [into register x] (synonym for

> 			yy, |linewise|). This is the builtin mapping, see
> 			|Y-default| for neovim's default mapping.

Reviewer: - the builtin normal command is not a "mapping", it is a (normal) command.
- we never say "neovim" (lowercase) in the help docs or anywhere else, and also rarely "Neovim". Almost always the help docs say "Nvim". In this case there isn't a reason to mention Nvim at all.
    - I think we need a linter for this at this point...
- There is really no reason to say anything at all here, given that the very next line mentions the default mapped nature.
    - Might seem trivial, but we need a repeatable pattern that doesn't result in clunky text all over the docs.

```suggestion
			yy, |linewise|).
```

							*Y-default*

> ["x]Y                   This command maps to "y$". see |nvim-default-mappings|

Reviewer: - I do not understand why the `["x]Y` in column 1 is repeated.
- Don't need "see".
- Can be more terse without loss of clarity, and now we have a repeatable pattern.

```suggestion
["x]Y                   Maps to "y$" by default. |nvim-default-mappings|
```
